bundles/org.eclipse.equinox.preferences/META-INF/MANIFEST.MF

```diff
@@ -7,11 +7,11 @@ Bundle-Activator: org.eclipse.core.internal.preferences.Activator
 Bundle-Vendor: %providerName
 Bundle-Localization: plugin
 Require-Bundle: org.eclipse.equinox.common;bundle-version="[3.2.0,4.0.0)",
- org.eclipse.equinox.registry;bundle-version="[3.2.0,4.0.0)";resolution:=optional
+ org.eclipse.equinox.registry;bundle-version="[3.2.0,4.0.0)";resolution:=optional,
+ org.osgi.service.prefs;bundle-version="[1.1.0,1.2.0)";visibility:=reexport
```

Comment:
You need to remove the `Import-Package` for `org.osgi.service.prefs`.  I don't recall what the behavior is supposed to be if you require->reexport a bundle but then import a package that required->reexport bundle also exports.  What should happen if your import wires to a different export from what is required?  Anyway, you should just remove the `Import-Package` regardless.

Reply:
Done.
Btw. We had the same topic in PR 26: https://github.com/eclipse-equinox/equinox.bundles/pull/26#discussion_r859800539